docs/guides/waffle-testing.md

```diff
@@ -125,6 +125,23 @@ expect(await greeter.greet()).to.equal("Hola, mundo!");
 
 We can modify the state of a contract in the same way we read from it. Calling `setGreeting` will set a new greeting message. After the `Promise` is resolved, we perform another assertion to verify that the greeting change took effect.
 
+### Running tests in parallel
+
+You can run your tests in parallel by using the `--parallel` flag:
+
+```
+$ npx hardhat test --parallel
+```
+
+Most of the time, running your tests serially or in parallel should produce the same results, but there are some scenarios where tests run in parallel will behave differently:
+
+- In serial mode, all the test files share the same instance of the [Hardhat Runtime Environment](/advanced/hardhat-runtime-environment.html), but in parallel mode each test file will have its own fresh instance. For example, if one test file deploys a contract, then that contract won't exist in the other test files.
+- The `.only` modifier doesn't work in parallel mode. You can use [`--grep`](https://mochajs.org/#-grep-regexp-g-regexp) to run specific tests.
+- The duration of individual tests might be different, because parallel mode uses more system resources. There's a chance that some tests start timing out for that reason. You can increase the tests timeout if you run into this problem.
```

Comment:
```suggestion
- Because parallel mode uses more system resources, the duration of individual tests might be longer, so there's a chance that some tests start timing out for that reason. You can increase the tests timeout if you run into this problem.
```

Reply:
Applied the suggestion and added some links about increasing timeouts. Check fe479e063ef253c8180b2eab7ab9be86177d427d.